src/server.js

```diff
@@ -1,4 +1,5 @@
 import registerApiDocs from './swagger';
+
```

Comment:
Why this changes here

Reply:
Nothing changed I did rebase Develop, this file is the same as from Develop